lib/SparqlEndpointFetcher.ts

```diff
@@ -97,9 +97,9 @@ export class SparqlEndpointFetcher {
    * @see IBindings
    * @param {string} endpoint A SPARQL endpoint URL. (without the `?query=` suffix).
    * @param {string} query    A SPARQL query string.
-   * @return {Promise<NodeJS.ReadableStream>} A stream of {@link IBindings}.
+   * @return {Promise<Readable>} A stream of {@link IBindings}.
    */
-  public async fetchBindings(endpoint: string, query: string): Promise<NodeJS.ReadableStream> {
+  public async fetchBindings(endpoint: string, query: string): Promise<Readable> {
```

Comment:
I apologise, but I must backtrack my previous comment.
We'll want the return type here to remain `NodeJS.ReadableStream` to avoid breaking changes in the API. (but in practise, both should be usable interchangeably)

Reply:
Sounds reasonable. :slightly_smiling_face: I have reverted the changes now, leaving only the function-internal one (to replace the library) and a cast in `fetchTriples` after swapping the `Readable` to come from `readable-stream` (it was already returning that one, as opposed to `NodeJS.ReadableStream`).

Do the changes look acceptable now?